spec/factories/course_experience_points_records.rb

```diff
@@ -11,10 +11,15 @@
         build(:course_user, course: course, user: user)
     end
     points_awarded { rand(1..20) * 100 }
-    reason 'EXP for some event'
+    actable_type 'self::PointsAwardingDummy'
```

Comment:
This isn't a good idea. The polymorphic column is now in an indeterminate state. The type is specified but not the id